ansible/roles/iroha-docker/tasks/check.yml

```diff
@@ -0,0 +1,23 @@
+# tasks in this file check that role can be aplaid to hosts
+
+- block:
+  - name: check | check for deprecated varible
+    fail:
+      msg: "iroha_overlay_network is deprecated use: 'iroha_network'"
+    when: iroha_overlay_network is defined
+
+  - name: check | check iroha_hostnames length and iroha_replicas
+    fail:
+      msg: "'iroha_hostnames' length ({{ iroha_hostnames|length }}) should match the 'iroha_replicas' ({{ iroha_replicas }})"
```

Comment:
Should it match `iroha_replicas` or `iroha_replicas` multiplied by number of hosts in inventory list?

Reply:
`iroha_hostnames` are set per host. I add example 2 in readme now t should be more clear.